src/metrics/prometheus.js

```diff
@@ -0,0 +1,94 @@
+// @flow
+// Copyright (c) 2018 Jaeger Author.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under
+// the License.
+
+import { Counter as PromCounter, Gauge as PromGauge } from 'prom-client';
+
+let CounterPromWrapper = class {
```

Comment:
why `let ... class` instead of `class xxx`?

Reply:
There is no reason.
This class expression is just another way of defining a new class in es6.
If class declaration(class xxx) is preferable for coding style, I'll change.

Reply:
We're using normal `class xxx` syntax everywhere else.